tests/storage_pool/storage_pool_test.go

```diff
@@ -381,6 +384,21 @@ func poolResizeIsInProgress(poolToBeResized *api.StoragePool) bool {
 	return poolSizeHasBeenChanged
 }
 
+func getWaitCount(poolSize uint64) int {
```

Comment:
This methods used in two places, can we put into common lib ?

One more thing instead of waiting for particular period of time. Can we check another way, example if rebalance percentage doesn't change for 15 minutes then we can fail the operation..

Reply:
> This methods used in two places, can we put into common lib ?
> 
> One more thing instead of waiting for particular period of time. Can we check another way, example if rebalance percentage doesn't change for 15 minutes then we can fail the operation..

Moved this method common lib.

Reblance percnetange show only when we do for add disk not for resize disk,
As of now added log message to monitor mrssage log. 
Will monitor that for more run and take this check as enhancement for next PR